src/DonorPerfect.php

```diff
@@ -208,7 +207,8 @@ public function call(string $action, array $parameters)
         foreach ($parameters as $param => $value) {
             $value = trim($value);
 
-            if (is_numeric($value) && strpos($value, 'e') === false) {
+            if (is_numeric($value) && strpos($value, 'e') === false && strpos($value, '+') === false &&
+               $param != 'CardExpirationDate') {
```

Comment:
```suggestion
            if (is_numeric($value)
                && strpos($value, 'e') === false
                && strpos($value, '+') === false
                && $param != 'CardExpirationDate'
            ) {
```

Reply:
@RonAbarbanel could you provide me the value of CardExpirationDate that is causing issues being interpreted as a numeric and then what it should actually be processed as?

I'd like to take a stab at fixing that through recognition of the value, rather than the param name since there is likely other params with similar behaviour.